release_readme.rst

```diff
@@ -28,15 +28,17 @@ Package Contents
 Overview
 --------
 
-The F5® OpenStack LBaaSv1 plugin allows you to orchestrate BIG-IP® load balancing services – including virtual IPs, pools, device service groups, and health monitoring – in an OpenStack environment.
+The F5 OpenStack LBaaSv1 plugin allows you to orchestrate BIG-IP® Local
+Traffic Manager™ (LTM®) services – including virtual IPs, pools, device service groups, and health monitoring – in an OpenStack environment.
 
 Before You Begin
 ----------------
 
-You will need the following to use the F5® OpenStack LBaaSv1 plugin.
+You will need the following to use the F5 OpenStack LBaaSv1 plugin.
+
+-  Licensed BIG-IP (hardware or virtual edition)
+-  OpenStack |openstack| Neutron network deployment
```

Comment:
In docs/conf.py, the openstack release version is set to 'Icehouse - Kilo'. It's correct to use the variable here instead of hard-coding it.